plugins/inputs/exec/exec_test.go

```diff
@@ -97,6 +98,7 @@ func TestExec(t *testing.T) {
 		MetricName: "exec",
 	})
 	e := &Exec{
+		log:      models.Logger{},
```

Comment:
Let's make a testutil.Logger, I don't want the models package used directly in any plugins or tests.  We could even make model.Logger unexported, but I care less about this so long as there are no examples to be copy/pasted.